bundle/src/test/java/com/adobe/acs/commons/models/injectors/impl/ChildResourceFromRequestInjectorTest.java

```diff
@@ -34,13 +33,14 @@
 import javax.jcr.RepositoryException;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertSame;
 
 public class ChildResourceFromRequestInjectorTest {
+
+    private ActivatorHelper activatorHelper = new ActivatorHelper();
+
     @Rule
-    public final AemContext context = new AemContext(ResourceResolverType.JCR_MOCK);
+    public final AemContext context = new AemContext(activatorHelper.afterSetup(), activatorHelper.beforeTeardown(), ResourceResolverType.JCR_MOCK);
```

Comment:
@stefanseifert can you suggest a better way to do this? Basically I need to invoke these methods whenever the context is created. This approach seems to work fine, but it seems a bit inelegant. I thought about creating a custom subclass of `AemContextImpl` but that would mean a fair bit of duplication from `AemContext`.